app/modules/ads/app-configuration.js

```diff
@@ -0,0 +1,33 @@
+export default class AppConfiguration {
+  load(instantGlobals = {}) {
+    const { instantConfig } = window.Wikia.adEngine;
+
+    this.instantGlobals = instantGlobals;
+
+    return instantConfig.getConfig().then((config) => {
+      this.config = config;
+
+      return config;
+    });
+  }
+
+  get(key, defaultValue = null) {
```

Comment:
I see you have smuggled this `defaultValue` argument. I know it is related to this additional type check, but still I have mixed feelings about it. We had similar discussion about `AdEngine.Context` (you, Jacek and me), and I believe we had agreed that `||` operator is more verbose and does not require knowledge about the definition of function. But again it creates problems with falsy values... I don't know, perhaps it is ok. I would like to here what Jacek has to say since he participated in the original discussion as well @jacekbj

Reply:
Yes we did and we decided to use lodash in app/ae3. However it's not available in mobile-wiki and I don't want to include lodash in the ember build. Falsy values are not ok :(. We need to have possibility to define falsy values from instant config.